pulp_smash/pulp3/pytest_plugin/__init__.py

```diff
@@ -285,6 +341,11 @@ def task_schedules_api_client(pulpcore_client):
         return TaskSchedulesApi(pulpcore_client)
 
 
+@pytest.fixture
```

Comment:
```suggestion
@pytest.fixture(scope="session")
```

Reply:
I can address it in a follow-up PR if needed.

Reply:
@goosemania you are free now, I can do a follow-up PR if needed

Reply:
Yes, and no. Yes, you only create (and tear down) the fixture once per session. But what is "the fixture"? In this case it is an instance of `StatusApi` preloaded with the `pulpcore_client` containing hostname, username and password. This object then has methods that can be called by tests and they will perform http requests on the spot.
Assuming these objects behave nicely in threads with our parallel testing strategy, scoping them by the session is the way for ultimate reusing.

Reply:
In the case of the service restart, the fixture to be cached is the function that will restart the services. And it can be run multiple times. Restarting services however is not meant for parallel tests.